Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -574,6 +574,13 @@ class SearchViewController: SiteTableViewController,
             }
         case .firefoxSuggestions:
             let firefoxSuggestion = firefoxSuggestions[indexPath.row]
+            if let clickInfo = firefoxSuggestion.clickInfo {
+                let position = SearchListSection.allCases.prefix { $0.rawValue < indexPath.section }
```

Comment:
```suggestion
                let position = SearchListSection.allCases.filter { $0.rawValue < indexPath.section }
```

Let's use `filter` here—I think it's a bit easier to understand, and does the same amount of work, since the `firefoxSuggestions` section is (for now) the last one.